modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_client.cc

```diff
@@ -61,45 +62,10 @@ void GrpcClientImpl::SendMsgToGrpc(const std::shared_ptr<CarStatus> &msg) {
   std::chrono::duration<double> time_used = end - start;
 
   // response check: error_code 4: time out; 0: success;
-  if (status.error_code() != 0) {
-    AERROR << "stub PushCarStatus error : code " << status.error_code();
-  } else {
-    ADEBUG << "stub PushCarStatus success. time used: "
-           << time_used.count() * 1000 << "ms"
-           << "error : code " << status.error_code();
-  }
-}
-
-void GrpcClientImpl::SendMsgToGrpc(
-    const std::shared_ptr<PerceptionObstacles> &msg) {
-  // verify perception obstacles msg valid
-  if (msg->perception_obstacle().empty()) {
-    AERROR << "SendObstaclesToObu msg is not valid";
-    return;
-  }
-
-  // set timeout
-  ClientContext context;
-  gpr_timespec timespec;
-  timespec.tv_sec = 0;
-  timespec.tv_nsec = 80000000;  // 80ms
-  timespec.clock_type = GPR_TIMESPAN;
-  context.set_deadline(timespec);
-  UpdateStatus response;
-
-  // time used statistics
-  auto start = std::chrono::steady_clock::now();
-  Status status = stub_->PushPerceptionResult(&context, *msg, &response);
-  auto end = std::chrono::steady_clock::now();
-  std::chrono::duration<double> time_used = end - start;
-
-  // response check: error_code 4: time out; 0: success;
-  if (status.error_code() != 0) {
-    AERROR << "stub PushPerceptionResult error : code " << status.error_code();
-  } else {
-    ADEBUG << "stub PushPerceptionResult success. time used: "
-           << time_used.count() * 1000 << "ms";
-  }
+  AINFO << "stub PushCarStatus Time used: " << time_used.count() * 1000
+        << " ms";
+  LOG_IF(ERROR, status.error_code() != 0)
```

Comment:
Please don't use LOG_IF, RETURN_IF, BREAK_IF, CONTINUE_IF stuffs. It hurts readability.